h/views/api/annotations.py

```diff
@@ -166,6 +167,32 @@ def delete(context, request):
     return {"id": context.annotation.id, "deleted": True}
 
 
+@api_config(
+    versions=["v1", "v2"],
+    route_name="api.annotation.reindex",
+    request_method="POST",
+    permission=Permission.Annotation.UPDATE,
+    link_name="annotation.reindex",
+    description="Reindex an annotation",
+)
+def reindex(context, request):  # pragma: no cover
+    """
+    Reindex one particular annotation.
+
+    The main motivation for this endpoint is to support
+    the functional tests, which often need to make sure an annotation
+    created in the DB is up to date in ES.
+    """
+    if not request.registry.settings["h.dev"]:
```

Comment:
We only have one of these AFAICT, we could refactor it into a predicate if needed but for now I like the explicitness of it.